website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,220 @@
+
+![Filter bar example](/assets/patterns/filter-patterns/filter-bar-segmented-group.png =728x*)
+
+**Example of a horizontal filter bar paired with a data set expressed as a table.**
+
+![Horizontal orientation of the filter bar](/assets/patterns/filter-patterns/layout-filter-bar.png =559x*)
+
+Depending on the filter type and how they are related, multiple [Segmented Groups](/components/segmented-group) can be used to group similar filters together. For example; a [Search Input](/components/form/text-input#search) grouped with a [Dropdown](/components/dropdown) to limit the filter parameters that are included in the search.
+
+**Limiting the returned search results to a specific parameter**
+
+![Searching across a specific parameter](/assets/patterns/filter-patterns/filter-bar-similar-filters.png =718x*)
+
+#### Sidebar
+
+If space permits, filters can be positioned vertically in a **sidebar** at the page level or in line with the data set.
+
+##### Page-level sidebar
+
+Page-level sidebars can be used for more complex data sets and if filtering content of the entire page is required and can be oriented on the left side of the viewport.
+
+![Vertical orientation of the filter bar](/assets/patterns/filter-patterns/layout-sidebar-left-page-level.png =559x*)
+
+##### In-line sidebar
+
+In-line sidebars can be used when the number of filterable parameters is minimal and if space permits.
+
+![Vertical orientation of the filter bar on the left](/assets/patterns/filter-patterns/layout-sidebar-left-inline.png =559x*)
+
+### Applied filters
+
+Display **applied filters** and provide the user with a method to clear all filters at once. Applied filters should be represented using a [Tag](/components/tag) component, which allows for the individual dismissal of a filter value.
+
+![Applied filters](/assets/patterns/filter-patterns/applied-filters.png =450x*)
+
+#### Positioning with filter bars
+
+Applied filters should be positioned between the data set and the filter bar with a `16px` gap between elements.
+
+![Applied filters with filter bar positioning](/assets/patterns/filter-patterns/applied-filters-positioning-filter-bar.png =559x*)
+
+In the case of a filter sidebar, the applied filters should be positioned directly above the dataset with a `16px` gap.
+
+![Applied filters with filter sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar.png =559x*)
+
+If the filters are positioned in a page-level sidebar, sometimes displaying the applied filters isn't necessary because they are implied in the sidebar. An action to clear all of the filters should be included within the sidebar itself.
+
+![Applied filters with clear all in the sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar-clear-all.png =559x*)
```

Comment:
This section doesn't feel helpful for a consumer because these contradict one another. In the first example, you mention that they should show the applied filters above the dataset with a clear button at the end. In the second example, you mention that sometimes that isn't necessary, and the clear button should be in the sidebar. 

Let's lean into external research here and make an opinionated decision. We recommend that they either always show them or don't show them and then clarify why. We can always revise that decision if/when we get user feedback.

Reply:
Ok the opinionated decision here is to always display them above the data set; this is currently what Developer does and I think it's a the most explicit method to 1) maintain consistency across the different filter orientations and 2) maintain consistency with the single filter clear and the bulk clear.